examples/asio-grpc/CMakeLists.txt

```
@@ -16,3 +16,20 @@ find_package(asio-grpc CONFIG REQUIRED)
 add_executable(boo boo.cpp)
 target_link_libraries(boo PUBLIC asio-grpc::asio-grpc)
 # DOCUMENTATION_END }
+
+# Test double library creation
+find_package(asio-grpc CONFIG REQUIRED)
+
+# Check license
+string(COMPARE EQUAL "${asio-grpc_LICENSES}" "" is_empty)
+if(is_empty)
+  message(FATAL_ERROR "Licenses not found")
+endif()
+
+message("asio-grpc licenses:")
+foreach(x ${asio-grpc_LICENSES})
+  message("* ${x}")
+  if(NOT EXISTS "${x}")
+    message(FATAL_ERROR "File not found")
+  endif()
+endforeach()
```

small nitpick, non-blocking: please add a newline at the end of files

```suggestion
endforeach()

```